test/run_dependency_with_broken_root/README.md

```diff
@@ -0,0 +1,25 @@
+# run_dependency_with_broken_root
+
+[![Package Version](https://img.shields.io/hexpm/v/run_dependency_with_broken_root)](https://hex.pm/packages/run_dependency_with_broken_root)
+[![Hex Docs](https://img.shields.io/badge/hex-docs-ffaff3)](https://hexdocs.pm/run_dependency_with_broken_root/)
+
+```sh
+gleam add run_dependency_with_broken_root
+```
+```gleam
+import run_dependency_with_broken_root
+
+pub fn main() {
+  // TODO: An example of the project in use
+}
+```
+
+Further documentation can be found at <https://hexdocs.pm/run_dependency_with_broken_root>.
+
+## Development
+
+```sh
+gleam run   # Run the project
+gleam test  # Run the tests
+gleam shell # Run an Erlang shell
+```
```

Comment:
Could you document what this package is for here please 🙏 

Also could you move this from `test/` to `test/fixtures/` or something as this this package is not a test that can be run- it'll always fail.